bindings/c/test/mako/mako.c

```diff
@@ -212,17 +218,40 @@ int populate(FDBTransaction *transaction, mako_args_t *args, int worker_id,
         xacts = 0;
         timer_prev.tv_sec = timer_now.tv_sec;
         timer_prev.tv_nsec = timer_now.tv_nsec;
+
+	/* enable transaction tracing */
+	if (args->txntrace) {
+	  tracetimer++;
+	  if (tracetimer == args->txntrace) {
+	    fdb_error_t err;
+	    tracetimer = 0;
+	    fprintf(debugme, "DEBUG: txn tracing %s\n", keystr);
+	    err = fdb_transaction_set_option(transaction,
+					     FDB_TR_OPTION_DEBUG_TRANSACTION_IDENTIFIER,
+					     (uint8_t *)keystr, strlen(keystr));
+	    if (err) {
+	      fprintf(stderr,
+		      "ERROR: fdb_transaction_set_option(FDB_TR_OPTION_DEBUG_TRANSACTION_IDENTIFIER): %s\n",
+		      fdb_get_error(err));
+	    }
+	    fdb_transaction_set_option(transaction,
```

Comment:
I think this needs `err = ` as well. I would make the suggestion, but you might also want to adjust the alignment of the next two lines at the same time.

Reply:
Oh yes, it does.  Fixed in the next commit.